boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20-memory_map.dtsi

```diff
@@ -187,8 +187,8 @@
 		#address-cells = <1>;
 		#size-cells = <1>;
 
-		storage_partition: partition@136000 {
-			reg = <0x136000 DT_SIZE_K(24)>;
+		storage_partition: partition@1e3000 {
```

Comment:
This change the assumption that the DFU partition is the last one in the Application's area.
Need to discuss this if this change does not break one of the update scenarios (w/o TFm, w/o A/B type updates, w/o SDFU bundles).

Reply:
For Wi-Fi we don't use DFU, so, I need someone to test/comment if DFU works with this changes or not? @ArekBalysNordic I guess Matter uses DFU, so, will you be willing to test?

Reply:
Currently, testing is not possible since we haven't run Matter on nrf54H20. Anyway, we will need to redefine those partitions in our samples' DTS files so there is no matter what is going to be in the dtsi file. But I guess, testing it using existing DFU tests should be enough.